templates/widgets/job-output.hamlet

```diff
@@ -1,11 +1,27 @@
-$maybe x <- jobStdout job
-  $if not (null x)
-    <pre .logs .stdout>
-      $forall ln <- lines x
-        <span .prefix>stdout</span>^{colorizedLogLine ln}
+$case output
+  $of JobOutputInProgress (Entity jobId job)
+    <pre
+      .logs-stream
+      ##{streamElementId}
+      data-stream-url=@{repoP (jobOwner job) (jobRepo job) $ jobLogR jobId}
+      data-stream-loaded=false
+      >
+      Fetching job log...
 
-$maybe x <- jobStderr job
-  $if not (null x)
-    <pre .logs .stderr>
-      $forall ln <- lines x
-        <span .prefix>stderr</span>^{colorizedLogLine ln}
+  $of JobOutputCompleted logLines
+    <pre>
+      $forall logLine <- logLines
+        ^{colorizedJobLogLine logLine}
+
+  $of JobOutputLegacy job
```

Comment:
We could migrate the legacy output columns to job-log-lines:

- Write all Stdout lines as of `t, t+1s, t+2s, ..., t+Ns`, then
- Write all Stderr lines as of `t+(N+1)s, t+(N+2)s, ...`